specifications/cdsc-wg1-01.md

```diff
@@ -1 +1,292 @@
+
+<a id="key-words" href="#key-words" class="permalink">🔗</a> Key Words: "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are defined in accordance with [BCP 14](https://www.rfc-editor.org/info/bcp14).
+
+## 3. Metadata Endpoint <a id="metadata-endpoint" href="#metadata-endpoint" class="permalink">🔗</a>
+
+Servers supporting this specification SHALL offer one or more HTTPS URLs from which a Client can perform a GET request to retrieve Server metadata, and the Server will respond with the Server's metadata JSON object.
+
+For successful responses, Servers MUST respond with a "200 OK" Status Code and body containing a JSON object that is formatted as defined in [Metadata Object Format](#metadata-object-format). Cache-optimized responses, such as "304 Not Modified", to Client requests with cached value headers, such as `If-None-Match`, is also acceptable.
+
+For invalid requests and error responses, Servers MUST respond with the appropriate Response Code for the type of issue or error (e.g. "503 Service Unavailable" if the Server is temporarily down for maintenance).
+
+### 3.1. Metadata Well-Known URI <a id="metadata-well-known-uri" href="#metadata-well-known-uri" class="permalink">🔗</a>
+
+If the Server wishes to provide metadata publicly, the Server MUST support a metadata endpoint at the Well-Known URI `"/.well-known/carbon-data-spec.json"` on their hostname (e.g. https://example.com/.well-known/carbon-data-spec.json).
+
+Servers MAY additionally support other public and restricted URLs for special case or private metadata endpoints, as needed.
+
+### 3.2. Metadata Object Format <a id="metadata-object-format" href="#metadata-object-format" class="permalink">🔗</a>
+
+Metadata objects are formatted as JSON objects and contain named values, some of which are required and some of which are optional. The following values are included in the default list available in metadata objects.
+
+* `cds_metadata_version` - _string_ - (REQUIRED) The version of metadata object format this Server follows, which for this version of the specification is `v1`
+* `cds_metadata_url` - _string_ - (REQUIRED) A URL link to this metadata object's canonical network location
+* `created` - _ISO8601 datetime_ - (REQUIRED) When the metadata url was first made available
+* `updated` - _ISO8601 datetime_ - (REQUIRED) When this metadata object was last modified
```

Comment:
From @jhughes22 ([on May 25, 2023](https://github.com/carbon-data-specification/Customer-Data/pull/23#discussion_r1205737110)):

---

Are zoned times allowed or are timestamps all in UTC? ISO8601 allows for either.

EDIT: I agree with the general suggestion in Greg's comment to be very explicit when it comes to timestamps

---

Reply:
@jhughes22 Updated to specify `datetime` as RFC 3339, which requires a timezone offset.